comp/core/tagger/tagger.go

```diff
@@ -365,8 +367,14 @@ func (t *TaggerClient) ResetCaptureTagger() {
 func (t *TaggerClient) EnrichTags(tb tagset.TagsAccumulator, originInfo taggertypes.OriginInfo) {
 	cardinality := taggerCardinality(originInfo.Cardinality)
 
+	// If dogstatsdUnifiedEnabled is disabled, we use DogStatsD's Legacy Origin Detection.
+	// TODO: remove this when dogstatsd_origin_detection_unified is enabled by default
+	if !t.datadogConfig.originDetectionUnifiedEnabled {
+		originInfo.ProductOrigin = taggertypes.ProductOriginDogStatsDLegacy
+	}
+
 	switch originInfo.ProductOrigin {
-	case taggertypes.ProductOriginDogStatsD:
+	case taggertypes.ProductOriginDogStatsDLegacy:
```

Comment:
**❓ question:** ‏just to make sure, is the `default` case what you consider as the "unified" way, or will the "unified" way be implemented later ?

Reply:
The unified setting will make sure that every Origin Detection uses the `default` case. In the future, we will remove the `ProductOriginDogStatsDLegacy` case in favor of just having the `default` case.